scripts/mintGen.js

```diff
@@ -8,7 +8,9 @@ async function main() {
   const DAOToken = await Arc.Utils.requireContract('DAOToken');
   const genToken = await DAOToken.at(genTokenAddress)
 
-  await Promise.all(web3.eth.accounts.map(async account => {
+  const daoAddress = '0xf7b7be05d6c115184f78226f905b643dd577fa6b';
```

Comment:
this is the ganacheDb DAO right? what if they don't use ganacheDb? I guess we just encourage that